components/submissions/SubmissionTextBox.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC } from "react"
+
+interface Props {
+    title: string,
+    subtitle: string,
+}
+
+export const SubmissionTextBox:FC<Props> = ({ title, subtitle }) => {
+    return (
+        <div className='flex flex-col mb-8'>
```

Comment:
you're using single quotes for props here but double quotes on other files. I'd recommend configuring prettier on this repo so this kind of thing is automatically handled on saving the file and you don't need to worry about small stuff like this